fedora-toolbox

```diff
@@ -262,6 +264,64 @@ enter()
 )
 
 
+run_rm()
+(
+    if [ $1 = 1 ]; then
+        if ! rm_output=$($prefix_sudo podman $2 $3 --force 2>&42); then
+            echo "$0: failed to remove $3"
+            exit 1
+        fi
+    else
+        if ! rm_output=$($prefix_sudo podman $2 $3 2>&42); then
+            echo "$0: failed to remove $3"
+            exit 1
+        fi
+    fi
+    if ! [[ $rm_output = "" ]]; then
+        echo -e "${LRC}$4 $rm_output removed${NC}"
+    fi
+)
+
+
+remove()
+(
+    if [ $4 = "rm" ]; then
+        keyword="container"
```

Comment:
I wonder if this will be better to split this into separate `remove_images` and `remove_containers` functions. It's nice that you have managed to avoid duplicated code by factoring out the common parts, but it makes the final commands slightly harder to read. Given that we are dealing with shell scripts, it might be better to optimize for readability. I don't know.

At least, I'd suggest copying the function parameters (ie. $1, $2, etc.) into variables with more descriptive names at the top of the function.

Reply:
There are now two separate functions - remove_image and remove_container. Function parameters are also copied to variables for better readability.